### PR TITLE
Append to file to allow catching all configuration when the function is executed more than once

### PR DESCRIPTION
**Pull request by jbliznak**
https://issues.jboss.org/browse/WFWIP-214

Simply replace the file redirection with append to the file so that configuration of all drivers is saved in drivers.cli

### CODE REVIEW

**Comment by jfdenise** [comment]
@yersan , this seems an interesting fix.

File: jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/install-common/install-common.sh [hunk 1]
```diff
@@ -106,7 +106,7 @@ function configure_drivers(){
         if [ "${configMode}" = "xml" ]; then
           sed -i "s|<!-- ##DRIVERS## -->|${drivers}<!-- ##DRIVERS## -->|" $CONFIG_FILE
         elif [ "${configMode}" = "cli" ]; then
-          echo "${drivers}" > ${S2I_CLI_DRIVERS_FILE}
+          echo "${drivers}" >> ${S2I_CLI_DRIVERS_FILE}
```
Comment (jfdenise) [inline jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/install-common/install-common.sh]: @yersan , @jbliznak  we can have an issue if the CONFIG_ADJUSTMENT_MODE == cli, the cli scipt will be executed multiple time, so the n-1 driver will be re-executed when adding the driver n. There is no clearing of S2I_CLI_DRIVERS_FILE.

Comment (yersan) [reply jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/install-common/install-common.sh]: I agree. It seems we have to append the CLI operations here always but find a better place to invoke the exec_cli_scripts "${S2I_CLI_DRIVERS_FILE}" if we are in pure cli mode, probably after invoking the install.sh script

**Comment by jfdenise** [comment]
@yersan , why don't we delay in all cases? cli or xml_cli?

**Comment by yersan** [comment]
@jfdenise Config mode could be "cli" if  is not found, but we could be in the case where the desired configuration is xml_cli. That means the user wants markers replacements if they are, and if not, try by cli. Here we are in S2I phase and executed the embedded server here will delete all the markers.

**Comment by yersan** [comment]
@jfdenise now I see your point.

We can delay in all cases, however, the idea was to do as much as we can in S2I phase.
Delay in all cases is the easiest solution here, and we work fine too.

**Comment by jfdenise** [comment]
@yersan , I agree, I am not fully sure but I am thinking that having a single behavior is simpler. Adding few lines to the CLI script is not that bad, what kills us is JVM startup.

**Comment by jbliznak** [comment]
So guys, should I change something? I admit I still don't have the whole picture about how all these scripts play together so it is very likely I didn't think about all possible execution branches. In any case feel free to close this if you think there is more that needs to be done.

**Comment by jfdenise** [comment]
@jbliznak , I just opened https://github.com/wildfly/wildfly-cekit-modules/pull/94 tha twill replace yours. Really a simple adjustment to the execution context.

**Pull request closed without merging**